test/e2e/framework/node/helper.go

```diff
@@ -166,3 +165,13 @@ func taintExists(taints []v1.Taint, taintToFind *v1.Taint) bool {
 	}
 	return false
 }
+
+// IsARM64 checks whether the k8s Node has arm64 arch.
+func IsARM64(node *v1.Node) bool {
```

Comment:
I'm wondering whether a more general `IsArch` would be useful. But we can add thatlater  when there is a need.

/approve